guides/contexts.md

```diff
@@ -109,3 +109,13 @@ You can even refer directly to keys in the eventual return value of a Promise, e
 }
   </dust-demo-json>
 </dust-demo>
+
+### Streams
+
+As of Dust 2.7.0, Dust can read from Node-like Streams in the context. When Dust finds a Stream, it attaches `data`, `error`, and `end` listeners.
```

Comment:
Could you move Dust 2.7.0 to the `Streams` header, since all of these features are specific to 2.7.0.